From 4c815d4e9cea57f20918f3348b84d11f7ce50763 Mon Sep 17 00:00:00 2001
From: Ferran Aran <ferran@ferranaran.com>
Date: Wed, 8 Jan 2025 15:46:20 +0100
Subject: [PATCH] feat: replace by fuzzel

---
 wofi-pass | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/wofi-pass b/wofi-pass
index 90fa1d6..ae52e0c 100644
--- a/wofi-pass
+++ b/wofi-pass
@@ -11,7 +11,7 @@ FLAG_ONLYPASSWORD=0
 FLAG_SQUASH=0
 FLAG_TYPE=0
 FLAG_HELP=0
-FLAG_HOTKEYS_SUPPORTED=1
+FLAG_HOTKEYS_SUPPORTED=0
 FLAG_CASE_INSENSITIVE=0
 
 CMD_COPY="${CMD_COPY:-"wl-copy"}"
@@ -36,7 +36,7 @@ HOTKEY_OTP_RET=13
 FIELD_OTP_TAG="<<--OTP-->>"
 
 function _wofi() {
-    wofi "${@}"
+    fuzzel "${@}"
 }
 
 function _pass() {
-- 
2.47.0

